tests/all-in-one.sh

```diff
@@ -52,9 +52,69 @@ git-annex copy * --to GA-rclone-CI
 git-annex drop *
 git-annex get *
 
+# Test REMOVE with mocked git-annex
+mkdir -p $RCLONE_PREFIX/abc/def
+echo test > $RCLONE_PREFIX/abc/def/test
+[[ -f $RCLONE_PREFIX/abc/def/test ]]
+mock-git-annex <<EOF
+< ^VERSION
```

Comment:
I thought to suggest moving such scripts into some `tests/test-remove.mock` and then just `cat`ing them here. Since you would need path, you could get it from `readlink -f "$0" | xargs dirname` and avoid modifying `PATH` outside so you could run this `mock-git-annex`, but damn OSX lacks coreutils readlink but there are workarounds 
https://stackoverflow.com/questions/3572030/bash-script-absolute-path-with-os-x if you like idea/challenge... if not -- I guess we could keep them `as is` even if at least for now.

Reply:
I tried, but `mock-git-annex` would need to support variable substitution because of the `$RCLONE_PREFIX` in the heredoc, using some ugly sed or something nicer like [envsubst](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html). Seems fun to investigate but I'm not sure I have the time right now.